### PR TITLE
Make labels key type as a sequence.

### PR DESCRIPTION
**Pull request by rupakg**
[Fixes #95446514]

### CODE REVIEW

File: test/spec/directives/service-definition-edit.js [hunk 1]
```diff
@@ -785,8 +785,8 @@ describe('Directive: serviceDefinitionEdit', function () {
         });
       });
 
-      ['links', 'external_links', 'ports', 'volumes', 'environment'].forEach(function (key) {
-        describe('when the key (' + key + ') represents a string value', function () {
+      ['links', 'external_links', 'ports', 'expose', 'volumes', 'volumes_from', 'environment', 'env_file', 'dns', 'cap_add', 'cap_drop', 'dns_search', 'labels'].forEach(function (key) {
+        describe('when the key (' + key + ') represents a sequence value', function () {
           it('returns an empty array', function () {
             var result = element.isolateScope().createNewEmptyValueForKey(key);
             expect(result).toEqual(['']);
```
Comment (dharmamike) [inline test/spec/directives/service-definition-edit.js]: What would happen if this were actually an empty array?
`expect(result).toEqual([]);`

Comment (rupakg) [inline test/spec/directives/service-definition-edit.js]: the desc. should be 'returns an array with an empty string'

**Comment by dharmamike** [comment]
:+1:

**Comment by alexwelch** [comment]
:+1: code looks good

**Pull request merged**